ApiBundle/Controller/ContentController.php

```diff
@@ -101,10 +101,32 @@ public function showOrCreateAction(Request $request, $contentId)
     public function listAction(Request $request)
     {
         $contentType = $request->get('content_type');
+        $columns = $request->get('columns');
```

Comment:
all right.

So in this case, I would only suggest to refactor the extraction of all the parameter of the request in an external method